core/controllers/domain_objects_validator_test.py

```diff
@@ -359,11 +358,24 @@ def test_correct_exp_state_domain_content(self) -> None:
                 '</oppia-noninteractive-image>').to_dict()
         }
 
-        self.assertEquals(
-            (
-                domain_objects_validator.
-                validate_exp_change_dict_and_return_exp_change_domain_obj(
-                    correct_change_dict).to_dict()
-            ),
-            correct_change_dict
-        )
+        expected_domain_obj = exp_domain.ExplorationChange.from_dict(
```

Comment:
Expected objects should be created without any logic. So, in this case, just make a completely new object from scratch, or alternatively do something like `self.assertEqual('...', returned_domain_obj.cmd)` where the '...' is hardcoded.

Reply:
Done